runners/flink/flink_runner.gradle

```diff
@@ -200,6 +200,7 @@ def createValidatesRunnerTask(Map m) {
       excludeCategories 'org.apache.beam.sdk.testing.UsesCommittedMetrics'
       if (config.streaming) {
         excludeCategories 'org.apache.beam.sdk.testing.UsesImpulse'
+        excludeCategories 'org.apache.beam.sdk.testing.UsesTestStreamWithMultipleStages'  // BEAM-8598
```

Comment:
Even without this, the test won't be executed because it is a `NeedsRunner` test, which Flink currently does not run.